main.js

```diff
@@ -35,7 +35,12 @@ window.addEventListener('load', (event) => {
 
     // TODO: shuffle the deck to be in a random order, see:
     // https://medium.com/@nitinpatel_20236/how-to-shuffle-correctly-shuffle-an-array-in-javascript-15ea3f84bfb
```

Comment:
Once `TODO`s are done, they should be removed.

Maybe keep the `shuffle the deck to be in a random order` part of the comment as explanation, because that what the for loop does is not obvious by inspection

Reply:
Yeah I agree